mesonbuild/utils/universal.py

```diff
@@ -756,6 +756,20 @@ class VcsData:
     rev_regex: str
     dep: str
     wc_dir: T.Optional[str] = None
+    repo_can_be_file: bool = False
+
+    def find(self, curdir: Path) -> bool:
```

Comment:
`find` is too general a name for this. It should be something like `is_repodir` to be more explicit about what it actually does.